src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Automatic Instrumentation Preview
+sidebar_order: 10
+description: "Learn what transactions are captured after tracing is enabled."
+redirect_from:
+  - /performance/included-instrumentation/
```

Comment:
I don't believe we need a redirect for this page?

Reply:
Ups a good catch. No, we don't need that. Was a copy-paste error.